package.json

```diff
@@ -7,20 +7,20 @@
   "license": "MIT",
   "main": "lib",
   "scripts": {
-    "test": "mocha  --compilers js:babel-register test",
+    "test": "mocha --require babel-core/register test",
```

Comment:
When installing mocha v5 and running `npm test`, mocha logged a deprecation warning:

```
(node:23482) DeprecationWarning: "--compilers" will be removed in a future version of Mocha; see https://git.io/vdcSr for more info
```

That git.io link has a section for updating how to migrate from an older to the latest version of mocha with Babel 6 (see [here](https://github.com/mochajs/mocha/wiki/compilers-deprecation#what-should-i-use-instead-then)).

I removed `babel-register` and installed `babel-core` to support this code change.